### PR TITLE
Initial commit of the grub 2 role

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+- name: Ensure grub ins installed
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]:
```suggestion
- name: Ensure grub is installed
```

File: templates/default.j2 [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# {{ ansible_managed }}
+
+# Distribution defaults, can be overridden below
+GRUB_DEFAULT=0
+GRUB_TIMEOUT=5
+│GRUB_DISTRIBUTOR=`lsb_release -i -s 2> /dev/null || echo Debian`
```
Comment (SlothOfAnarchy) [inline templates/default.j2]: Why the pipe?

File: templates/default.j2 [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+GRUB_CMDLINE_LINUX_DEFAULT=""
+GRUB_CMDLINE_LINUX=""
```
Comment (SlothOfAnarchy) [inline templates/default.j2:8]: This should be configurable (for our "IPMI")

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: templates/default.j2 (hunk 3 above)
Comment (dasJ) [reply templates/default.j2:8]: Junge…

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**